metabase/metabase
# Hide custom GeoJson settings when they are set by the env var

Fixes https://github.com/metabase/metabase/issues/18862

How to test:
- `MB_CUSTOM_GEOJSON='{"864d0660-2e03-ba4e-f23c-7d31eb6ac7b8":{"name":"Custom States","url":"https://raw.githubusercontent.com/metabase/metabase/master/resources/frontend_client/app/assets/geojson/us-states.json","region_key":"STATE","region_name":"NAME"}}' yarn dev`
- Go to Admin > Maps. The page should open and it shouldn't be possible to view/edit custom GeoJson settings.
